docs/reference/tab-widgets/inference-api/infer-api-ingest-pipeline.asciidoc

```diff
@@ -165,6 +165,32 @@ and the `output_field` that will contain the {infer} results.
 
 // end::azure-ai-studio[]
 
+// tag::google-vertex-ai[]
+
+[source,console]
+--------------------------------------------------
+PUT _ingest/pipeline/google_vertex_ai_embeddings
```

Comment:
Should we include pipeline in the name here? It's pretty close to the index name `google-vertex-ai-embeddings` and the same as the inference endpoint id. Maybe that could lead to confusion for people not too familiar with the concepts (inference endpoint vs pipeline vs index)